meilisearch/index.py

```diff
@@ -181,6 +183,35 @@ def get_update_status(self, update_id):
             )
         )
 
+
+    def wait_for_pending_update(self, update_id, timeout_in_ms=2000, interval_in_ms=10):
+        """Wait until MeiliSearch processes an update, and get its status
+
+        Parameters
+        ----------
+        update_id: int
+            identifier of the update to retrieve
+        timeout_in_ms (optional): int
+            time the function should wait before rising a TimeoutError
+        interval_in_ms (optional): int
+            time interval the function should wait (sleep) between requests
+        Returns
+        ----------
+        update: `list`
+            List of all enqueued and processed actions of the index.
+        """
```

Comment:
It should not return the list of all the processed actions but only the only we waited for.

Reply:
ERROR: copy_paste_error detected 🎉

Reply:
😂 😂